porch/repository/pkg/git/commit.go

```diff
@@ -333,17 +351,27 @@ func entrySortKey(e *object.TreeEntry) string {
 	return e.Name
 }
 
-func storeCommit(store storer.EncodedObjectStorer, parent plumbing.Hash, tree plumbing.Hash, message string) (plumbing.Hash, error) {
+func storeCommit(store storer.EncodedObjectStorer, parent plumbing.Hash, tree plumbing.Hash, author, message string) (plumbing.Hash, error) {
 	now := time.Now()
+	var authorName, authorEmail string
+	if author != "" {
+		// Authenticated user info only provides one value...
+		authorName = author
```

Comment:
It might be an idea to have a richer user info type, rather than a string.  `nil` is also a bit safer than `""` (misusing a nil is a panic)

Reply:
Good point. I'll turn it into a pointer.